packages/sanity-plugin-nrkno-iframe-preview/README.md

```diff
@@ -0,0 +1,166 @@
+# @nrk/sanity-plugin-nrkno-iframe-preview
+
+Liveupdated iframe preview component for Sanity Studio, compatible with any framework.
+
+Queries are executed by Sanity Studio and passed to the iframe. 
+This has the beneficial in that the render-app in the iframe
+does not need to listen for sanity client updates itself.
+
+![preview.png](./docs/images/preview.png)
+_Figure 1: IFramePreview with desktopMinWidth configured_
+
+The app running in the iframe should use `@nrk/nrkno-iframe-preview-api`, 
+which allows it to specify a GROQ-query for the Studio to execute.
+
+The result of the query will be sent to the iframe whenever the query
+revision matches the studio revision (ie. whenever the data changes).
+
+This means that the render app does not need to listen for changes itself, as
+data flows directly from the Studio via iframe.postMessage.
+
+# Installation
+In Sanity studio project run:
+
+`npx sanity install @nrk/sanity-plugin-nrkno-iframe-preview`
+
+This will install & add the plugin to sanity.json plugin array.
+
+## At-a-glance
+
+* Use IFramePreview component in studio structure view.
+    * Configure with render-app preview-url.
+    * Optionally configure desktopMinWidth.
+* Use [nrkno-sanity-iframe-preview-api](../nrkno-iframe-preview-api/README.md) in the render-app.
+    * Configure a groq-query (or just use document directly from Sanity Studio as is).
+* Enjoy live-updated preview in Studio, with queries executed by the Studio on behalf of the render app.
+
+## Usage
+Should be used as part of Sanity Studio StructureBuilder code:
+```ts
+import { IFramePreview } from '@nrk/sanity-plugin-nrkno-iframe-preview'
+
+S.view
+  .component(IFramePreview)
+  .options({
+      url: (doc: SanityDocument) => 'iframe-url', // (doc) => (string | Promise<string>)
+      mapDocument: (doc: SanityDocument) => ({ _id: doc._id }),
+      desktopMinWidth: 900,
```

Comment:
url burde vel være en gyldig url og ikke `iframe-url`?

Om det ikke skal skrives hva mapDocument og desktopMinWidth er kan de vel bare fjernes imo

Reply:
Mnja tja. Skal se hvordan det ser ut når jeg lager noen flere eksempler + lenker til jsdocs.

+1 for "https://iframe-url.example i guess